.ci/ci_job_flags.sh

```diff
@@ -103,12 +103,14 @@ case "${CI_JOB}" in
 		"CRI_CONTAINERD_K8S")
 			export KUBERNETES="yes"
 			;;
-		"CC_CRI_CONTAINERD")
+		"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD")
 			# Export any CC specific environment variables
 			export CCV0="yes"
-			#export SKOPEO=${SKOPEO:-}
 			export UMOCI=yes
 			export SECCOMP=yes
+			if [ "${CI_JOB}" == "CC_SKOPEO_CRI_CONTAINERD" ]; then
+				export SKOPEO=yes
+			fi
```

Comment:
Nit: This could be simplified:

```suggestion
			[ "${CI_JOB}" = 'CC_SKOPEO_CRI_CONTAINERD' ] && export SKOPEO=yes
```

Reply:
True. I will send a v2. Thanks @jodh-intel !

Reply:
hmmm maybe I should not make that change otherwise the statement will return 1 in case of `CI_JOB` != `CC_SKOPEO_CRI_CONTAINER`. Or am I missing something @jodh-intel ?